### DISCUSSION TITLE
Discussion about blog post: tailwind-disadvantages

### DISCUSSION THREAD

**Discussion by ScriptRaccoon** [ScriptRaccoon/portfolio-website#13] [Announcements]
Let's discuss about https://scriptraccoon.dev/blog/tailwind-disadvantages

**Comment by ScriptRaccoon** [comment]
Let me know what you think!

**Reply by Hatareg** [reply]
Great article! I totally agree with you. You describe exactly what the problems are, with examples making it very clear. I love CSS and I do not understand why people do not take the time to learn it. When you know CSS it is very easy to quickly style your HTML, much quicker than writing some obscure abbreviations. Especially liked your JS framework comparison: "Imagine someone came up with a JavaScript framework where no variables have names because it is too hard to come up with good names."
